lib/codegen/selection/generator.cc

```diff
@@ -457,19 +457,25 @@ std::tuple<Value*, Value*, Value*, Value*> generator::fp8x4_to_fp16x4(Value *in0
 }
 
 Value* generator::bf16_to_fp32(Value *in0){
```

Comment:
Oh that's interesting, does NVPTX fail on BF16 <-> FP32 conversions? I remember facing something similar

Reply:
NVPTX can't select bf16 load/store. And sm<80 devices can't recognize bf16 in ptx. So I cast bf16 to fp16 whenever we want to load/store bf16 elements.

Reply:
Oh, I meant: can NVPTX select bf16 <-> fp32 conversion?

Reply:
I think NVPTX cannot select bf16 <-> fp32 conversion :(